src/app/common/TagList.vue

```diff
@@ -100,4 +104,13 @@ function getRoute(tag: LabelValue): RouteLocationNamedRaw | undefined {
   flex-wrap: wrap;
   gap: $kui-space-40;
 }
+
+.kuma-badge {
+  background-color: $kui-color-background-decorative-purple-weakest !important;
+}
+
+.kuma-badge,
+.kuma-badge a {
+  color: $kui-color-text-decorative-purple !important;
+}
```

Comment:
Is there any way to avoid the `!important`s here?

Reply:
I changed this a bit to avoid the `!important` even though in this case it was a perfectly fine use: we really don’t want anything to be able to override these colors again. That’s what `!important` is for.